### PR TITLE
Allow Symfony 3.0

### PR DESCRIPTION
**Pull request by dantleech**

### CODE REVIEW

**Comment by dantleech** [comment]
Not so simple -- guess this might be a good time to release 1.0 and only support ~2.8

**Comment by dbu** [comment]
yep, go for symfony 2.8. 

maybe tag the current state as 0.9 to indicate its not a 1.0.0 and that it won't be supported going forward.

when used as phar, the symfony version is not that relevant (as long as people are using a maintained version of php and not 5.3 or 5.4)

File: composer.json [hunk 1]
```diff
@@ -2,28 +2,25 @@
     "name": "phpcr/phpcr-shell",
     "description": "Shell for PHPCR",
     "require": {
-        "symfony/console": "~2.4",
+        "symfony/console": "~2.4|~3.0",
```
Comment (dantleech) [inline composer.json]: 2.8

**Comment by dantleech** [comment]
@dbu 1.0.0-beta3 is already the "current" state, more or less.

File: src/PHPCR/Shell/PhpcrShell.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
 class PhpcrShell
 {
     const APP_NAME = 'PHPCRSH';
-    const APP_VERSION = 'dev-master';
+    const APP_VERSION = '1.0.0-beta4';
```
Comment (dantleech) [inline src/PHPCR/Shell/PhpcrShell.php]: upda

**Comment by dantleech** [comment]
Bam

**Comment by dbu** [comment]
\o/

**Pull request merged**